frontend/test/__support__/server-mocks/collection.ts

```diff
@@ -14,8 +14,23 @@ import {
 } from "metabase-lib/metadata/utils/saved-questions";
 import { PERMISSION_ERROR } from "./constants";
 
-export function setupCollectionsEndpoints(collections: Collection[]) {
-  fetchMock.get("path:/api/collection/root", ROOT_COLLECTION);
+export function setupCollectionsEndpoints({
```

Comment:
We have different functions to setup error cases, and the one for collections already exist - it's `setupUnauthorizedCollectionsEndpoints`. Please use the existing one.

Reply:
I think setup like this would be semantically misleading - after all we're not testing authorization here.
It'd be more appropriate to use a more generic error.

BTW `setupUnauthorizedCollectionsEndpoints` is unused.
I'll reuse it as suggested but first refactor it to allow more generic errors.

Reply:
> We have different functions to setup error cases,

Did just that: eb8e6377729da45557ee926f2c17bda8064563ce